<!--
 * @Description: 
 * @Author: lilongguang
 * @Date: 2022-02-24 12:38:36
 * @LastEditors: lilongguang
 * @LastEditTime: 2022-04-03 14:21:28
-->

[1]Yu, Lijia, Deepak Kumar Tanwar, Emanuel Diego S. Penha, Yuri I. Wolf, Eugene V. Koonin, and Malay Kumar Basu. 2019. ‘‘Grammar of Protein Domain Architectures.” Proceedings of the National Academy of Sciences 116 (9): 3636–45. 10.1073/pnas.1814684116.

[2]Boutet E, Lieberherr D, Tognolli M, Schneider M, Bairoch A. UniProtKB/Swiss- Prot: The manually annotated section of the uniprot knowledgebase. Methods Mol Biol 2007;406:89–112.

[3]Kudo, Taku. 2018. ‘‘Subword Regularization: Improving Neural Network Translation Models with Multiple Subword Candidates.” ArXiv:1804.10959 [Cs], April. http://arxiv.org/abs/1804.10959.

[4]Sennrich, Rico, Barry Haddow, and Alexandra Birch. 2016. ‘‘Neural Machine Translation of Rare Words with Subword Units.” In Proceedings of the 54th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers), 1715–25. Berlin, Germany: Association for Computational Linguistics. 10.18653/v1/P16-1162.

[5]Wang Yanbin, You Zhu-Hong, Yang Shan, Li Xiao, Jiang Tong-Hai, Zhou Xi. A high efficient biological language model for predicting protein-protein interactions. Cells 2019;8(2):122. https://doi.org/10.3390/cells8020122.

[6]Brandes N, Ofer D, Linial M. ASAP: A machine learning framework for local protein properties. Database 2016;2016. https://doi.org/10.1093/database/ baw133.

[7] Ofer, Dan, and Michal Linial. 2015. ‘‘ProFET: Feature Engineering Captures High-Level Protein Functions.” Bioinformatics (Oxford, England), June. 10. 1093/bioinformatics/btv345.

[8] Budowski-Tal, Inbal, Yuval Nov, and Rachel Kolodny. FragBag, an Accurate Representation of Protein Structure, Retrieves Structural Neighbors from the Entire PDB Quickly and Accurately. Proceedings of the National Academy of Sciences of the United States of America. 2010. 107 (8): 3481–86. 10.1073/ pnas.0914097107.

[9] Boutet E, Lieberherr D, Tognolli M, Schneider M, Bairoch A. UniProtKB/Swiss- Prot: The manually annotated section of the uniprot knowledgebase. Methods Mol Biol 2007;406:89–112.

[10]Joulin, Armand, Edouard Grave, Piotr Bojanowski, and Tomas Mikolov. 2016. ‘‘Bag of Tricks for Efficient Text Classification.” ArXiv:1607.01759 [Cs], August. http://arxiv.org/abs/1607.01759.

[11]Rao, Roshan, Nicholas Bhattacharya, Neil Thomas, Yan Duan, Xi Chen, John Canny, Pieter Abbeel, and Yun S. Song. 2019. ‘‘Evaluating Protein Transfer Learning with TAPE,” June. https://arxiv.org/abs/1906.08230.

[12]Yang, Kevin K, Zachary Wu, Claire N Bedbrook, and Frances H Arnold. 2018. ‘‘Learned Protein Embeddings for Machine Learning.” Edited by Jonathan Wren. Bioinformatics 34 (15): 2642–48. 10.1093/bioinformatics/bty178.

[13]Mikolov T, Chen K, Corrado G, Dean J. Distributed representations of words and phrases and their compositionality. Nips 2013;1–9. https://doi.org/ 10.1162/jmlr.2003.3.4-5.951.

[14]Pennington, Jeffrey, Richard Socher, and Christopher Manning. 2014. ‘‘Glove: Global Vectors for Word Representation.” In Proceedings of the 2014 Conference on Empirical Methods in Natural Language Processing (EMNLP), 1532–43. Doha, Qatar: Association for Computational Linguistics. 10.3115/ v1/D14-1162.

[15]Asgari Ehsaneddin, Mofrad Mohammad RK. Continuous Distributed representation of biological sequences for deep proteomics and genomics. PLoS ONE 2015;10(11). https://doi.org/10.1371/journal.pone.0141287.

[16]Peters, Matthew E., Mark Neumann, Mohit Iyyer, Matt Gardner, Christopher Clark, Kenton Lee, and Luke Zettlemoyer. 2018. ‘‘Deep Contextualized Word Representations.” ArXiv:1802.05365 [Cs], March. http://arxiv.org/abs/1802. 05365.

[17]McCann, Bryan, James Bradbury, Caiming Xiong, and Richard Socher. 2018. ‘‘Learned in Translation: Contextualized Word Vectors.” ArXiv:1708.00107 [Cs], June. http://arxiv.org/abs/1708.00107.

[18]Bileschi ML, Belanger D, Bryant D, Sanderson T, Brandon Carter D, Sculley MA, et al. Using deep learning to annotate the protein universe. BioRxiv 2019. https://doi.org/10.1101/626507.

[19]Elnaggar, Ahmed, Michael Heinzinger, Christian Dallago, Ghalia Rihawi, Yu Wang, Llion Jones, Tom Gibbs, et al. 2020. ‘‘ProtTrans: Towards Cracking the Language of Life’s Code Through Self-Supervised Deep Learning and High Performance Computing,” July. http://arxiv.org/abs/2007.06225.

[20]Littmann, Maria, Michael Heinzinger, Christian Dallago, Tobias Olenyi, and & Burkhard Rost. 2020. ‘‘Embeddings from Deep Learning Transfer GO Annotations beyond Homology.” BioRxiv, September, 2020.09.04.282814. 10.1101/2020.09.04.282814.

[21]Heinzinger M, Ahmed Elnaggar Yu, Wang CD, Nechaev D, Matthes F, Rost B. Modeling aspects of the language of life through transfer-learning protein sequences. BMC Bioinf 2019;20(1):1–17. https://doi.org/10.1186/s12859- 019-3220-8.

[22]Choromanski, Krzysztof, Valerii Likhosherstov, David Dohan, Xingyou Song,
Andreea Gane, Tamas Sarlos, Peter Hawkins, et al. 2020. ‘‘Rethinking Attention with Performers.” ArXiv:2009.14794 [Cs, Stat], September. http:// arxiv.org/abs/2009.14794.

[23]Zhang N, Bi Z, Liang X, et al. OntoProtein: Protein Pretraining With Gene Ontology Embedding[J]. arXiv preprint arXiv:2201.11147, 2022.

[24]逻辑回归算法参考论文(吴雪81)

[25]朴素贝叶斯参考论文(吴雪82)

[26]吴雪83，应用广泛

[27]决策树(吴雪84)

[28]KNN(吴雪85)

[29]支持向量机(吴雪90)

[30]Salwinski   L, Miller   CS, Smith   AJ, et al.   The database of interacting proteins: 2004 update. Nucleic Acids Res  2004;32(suppl_1):D449–51.

[31]Pazos F, Helmer-Citterich M, Ausiello G, et al. Correlated mutations contain information about protein-protein interaction[J]. Journal of molecular biology, 1997, 271(4): 511-523.

[32]Balakrishnan, Sivaraman et al. “Learning Generative Models for Protein Fold Families.” Proteins: Structure, Function, and Bioinformatics 79.4(2011): 1061–1078.

[33]Bosheng Song, Xiaoyan Luo, Xiaoli Luo, Yuansheng Liu, Zhangming Niu, Xiangxiang Zeng, Learning spatial structures of proteins improves protein–protein interaction prediction, Briefings in Bioinformatics, 2022;, bbab558, https://doi.org/10.1093/bib/bbab558

[34]1984. Nomenclature and Symbolism for Amino Acids and Peptides. European
Journal of Biochemistry 138, 1 (1984), 9–37. https://doi.org/10.1111/j.1432-1033.
1984.tb07877.x

[35]Muhao Chen, Chelsea J T Ju, Guangyu Zhou, Xuelu Chen, Tianran Zhang, KaiWei Chang, Carlo Zaniolo, and Wei Wang. 2019. Multifaceted protein-protein
interaction prediction based on Siamese residual RCNN. Bioinformatics 35, 14
(07 2019), i305–i314

[36]Muhao Chen, Chelsea J T Ju, Guangyu Zhou, Xuelu Chen, Tianran Zhang, KaiWei Chang, Carlo Zaniolo, and Wei Wang. 2019. Multifaceted protein-protein
interaction prediction based on Siamese residual RCNN. Bioinformatics 35, 14
(07 2019), i305–i314

[37]Rico Sennrich, Barry Haddow, and Alexandra Birch. 2016. Neural Machine
Translation of Rare Words with Subword Units. In Proceedings of the 54th Annual
Meeting of the Association for Computational Linguistics (Volume 1: Long Papers).
Association for Computational Linguistics, Berlin, Germany, 1715–1725

[38] Jacob Devlin, Ming-Wei Chang, Kenton Lee, and Kristina Toutanova. 2019. BERT:
Pre-training of Deep Bidirectional Transformers for Language Understanding. In
Proceedings of the 2019 Conference of the North American Chapter of the Association
for Computational Linguistics: Human Language Technologies, Volume 1 (Long and
Short Papers). Association for Computational Linguistics, Minneapolis, Minnesota,
4171–4186

[39]You Z-H, Zhu L, Zheng C-H, et al. Prediction of protein-protein interactions from amino acid sequences using a novel multi- scale continuous and discontinuous feature set. BMC Bioinfor- matics, 2014;15:S9. Springer.

[40]Wong L, You Z-H, Li S, et al. Detection of protein–protein interac- tions from amino acid sequences using a rotation forest model with a novel PR-LPQ descriptor. In: International Conference on Intelligent Computing. Fuzhou, China: Springer, 2015, 713–20.

[41]Yang L, Xia J-F, Gui J. Prediction of protein–protein interactions from protein sequence using local descriptors. Protein Pept Lett 2010;17(9):1085–90.

[42]You Z-H, Lei Y-K, Zhu Lin, et al.. Prediction of protein–protein interactions from amino acid sequences with ensemble extreme learning machines and principal component analysis. BMC Bioin- formatics 2013;14:S10. Springer.

[43]Xiuquan D, Sun S, Changlin H, et al. DeepPPI: boosting prediction of protein–protein interactions with deep neural networks. J Chem Inf Model 2017;57(6):1499–510.

[44]Sun T, Zhou B, Lai L, et al. Sequence-based prediction of protein- protein interaction using a deep-learning algorithm. BMC Bioin- formatics 2017;18(1):1–8.

[45]Li H, Gong X-J, Hua Y, et al. Deep neural network based predic- tions of protein interactions using primary sequences. Molecules 2018;23(8):1923.

[46]Ioannidis V N, Song X, Manchanda S, et al. Drkg-drug repurposing knowledge graph for covid-19[J]. arXiv preprint arXiv: 2010.09600, 2020.

[47] Bordes A, Usunier N, Garcia-Duran A, et al. Translating embeddings for modeling multi-relational data. In:Advances in neural information processing systems.2013, 2787-2795.

[48]Guo Y, Lezheng Y, Wen Z, et al. Using support vector machine combined with auto covariance to predict protein– protein interactions from protein sequences. Nucleic Acids Res 2008;36(9):3025–30.

[49]You Z-H, Li J, Gao X, et al. Detecting protein-protein interactions with a novel matrix-based protein sequence representation and support vector machines. Biomed Res Int 2015;2015:867516.

[50]You Z-H, Jian-Zhong Y, Zhu L, et al. A mapreduce based paral- lel svm for large-scale predicting protein–protein interactions. Neurocomputing 2014;145:37–43.

[51]Huang Y-A, You Z-H, Gao X, et al. Using weighted sparse repre- sentation model combined with discrete cosine transformation to predict protein–protein interactions from protein sequence. Biomed Res Int 2015;2015:902198.

[52]Wong L, You Z-H, Li S, et al. Detection of protein–protein interac- tions from amino acid sequences using a rotation forest model with a novel PR-LPQ descriptor. In: International Conference on Intelligent Computing. Fuzhou, China: Springer, 2015, 713–20.

[53]Xiuquan D, Sun S, Changlin H, et al. DeepPPI: boosting prediction of protein–protein interactions with deep neural networks. J Chem Inf Model 2017;57(6):1499–510.

[54]You Z-H, Lei Y-K, Zhu Lin, et al.. Prediction of protein–protein interactions from amino acid sequences with ensemble extreme learning machines and principal component analysis. BMC Bioin- formatics 2013;14:S10. Springer.

[55]Chen H, Li F, Wang L, et al. Systematic evaluation of machine learning methods for identifying human–pathogen protein– protein interactions. Brief Bioinform 2021;22(3):bbaa068.

[56]Laura Restrepo-Pérez, Chirlmin Joo, and Cees Dekker. 2018. Paving the way to
single-molecule protein sequencing. Nature nanotechnology 13, 9 (2018), 786–796.

[57]The UniProt Consortium. 2018. UniProt: a worldwide hub of protein knowledge.
Nucleic Acids Research 47, D1 (11 2018), D506–D515.

[58]Hashemifar S, Neyshabur B, Khan AA, et al. Predicting protein–
protein interactions through sequence-based deep learning.
Bioinformatics 2018;34(17):i802–10.

[59]Zagoruyko S, Komodakis N. Learning to compare image patches via convolutional neural networks[J]. computer vision and pattern recognition, 2015: 4353-4361.

[60]Li H, Gong X-J, Hua Y, et al. Deep neural network based predic-
tions of protein interactions using primary sequences. Molecules
2018;23(8):1923.

[61]Chen M, Ju CJ-T, Zhou G, et al. . Multifaceted protein–protein
interaction prediction based on siamese residual RCNNBioin-
formatics 2019;35(14):i305–14.

[62]Singh R, Park D, Xu J, et al. Struct2net: a web service to predict
protein–protein interactions using a structure-based approach.
Nucleic Acids Res 2010;38(suppl_2):W508–15.

[63]Qiangfeng Cliff Zhang, Donald Petrey, Lei Deng, Li Qiang,
Yu Shi, Chan Aye Thu, Brygida Bisikirska, Celine Lefebvre,
Domenico Accili, Tony Hunter, et al. Structure-based pre-
diction of protein–protein interactions on a genome-wide
scale. Nature, 490(7421):556–560, 2012.

[64]Mirabello C, Wallner B. InterPred: a pipeline to identify and
model protein–protein interactions. Proteins 2017;85(6):1159–70.

[65]Zhao C, Zang Y, Quan W, et al. Hiv1-human protein-protein
interaction prediction based on interface architecture sim-
ilarity. In: 2017 IEEE International Conference on Bioinformatics
and Biomedicine (BIBM), pp. 97–100. IEEE, 2017.

[66]Planas-Iglesias J, Bonet J, García-García J. Manuel A Marín-
López, Elisenda Feliu, and Baldo Oliva. Understanding pro-
tein–protein interactions using local structural features. J
Mol Biol 2013; 425(7): 1210–24.

[67]Zheng S, Rao J, Song Y, Zhang J, Xiao X, Fang EF, Yang Y, Niu Z. PharmKG: a dedicated knowledge graph benchmark for bomedical data mining. Brief Bioinform. 2021 Jul 20;22(4):bbaa344. doi: 10.1093/bib/bbaa344. PMID: 33341877.

[68]Percha B and Altman R B. A global network of biomedical relationships derived from text[J]. Bioinformatics, 2018, 34(15): 2614-2624

[69]Himmelstein D S and Baranzini S E. Heterogeneous network edge prediction: a data integration approach to prioritize disease-associated genes[J]. PLoS computational biology, 2015, 11(7): e1004259.

[70]Bordes A, Weston J, Collobert R, et al. Learning structured embeddings of knowledge bases[C]//Twenty-fifth AAAI conference on artificial intelligence. 2011.

[71]Wang Z, Zhang J, Feng J, et al. Knowledge graph embedding by translating on hyperplanes[C]//Proceedings of the AAAI Conference on Artificial Intelligence. 2014, 28(1).

[72]Trouillon T, Welbl J, Riedel S, et al. Complex embeddings for simple link prediction. International Conference on Machine Learning (ICML),2016.

[73]Qian W, Fu C, Zhu Y, et al. Translating Embeddings for Knowledge Graph Completion with Relation Attention Mechanism[C]//IJCAI. 2018: 4286-4292.

[74]Santos, A., Colaço, A.R., Nielsen, A.B. et al. A knowledge graph to interpret clinical proteomics data. Nat Biotechnol (2022).

[75]Binder, J., Ursu, O., Bologa, C. et al. Machine learning prediction and tau-based screening identifies potential Alzheimer’s disease genes relevant to immunity. Commun Biol 5, 125 (2022).